main.cpp

```diff
@@ -83,44 +106,82 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
-
+ */
+float eqChange(float gain, float frequency = 18.5f, bool bypass = false)
+{
+    ignoreUnused(gain, frequency, bypass);
+    return{};
+}
 /*
  2)
  */
-
+bool weatherConditions(int temperature, float humidity, bool isNight = true)
+{
+    ignoreUnused(temperature, humidity, isNight);
+    return {};
+}
 /*
  3)
  */
-
+void rating(float stars, int numbersOfReviews = 384)
+{
+    ignoreUnused(stars, numbersOfReviews);
+}
 /*
  4)
  */
-
+double rectangularRoomVolume(double length = 3.84, double width = 8.62, double height = 4.1)
```

Comment:
This function name does not conform to the programming guideline:
```
Name Things what they are.
Name Actions based on what they do
```
The name you chose reflects a property (noun) and does not contain a verb, which would reflect an action.
Functions perform work, so their names must contain a verb which describes the work being performed.
Examples of function names that contain verbs:
- `computeAValue()`
- `estimateDistanceTraveled()`
- `performDiagnostics()`
- [ ] revise